admin/tests/unit/controllers/edit-translation.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
What is up with all these no-unused-vars tags everywhere? Why does ESLint think we're not using vars? If it's a testing thing (I'm not going to comment everywhere, but this is littered throughout the testing code) maybe we should just disable this at the testing level?

Reply:
By any chance are you confusing this with `no-var` rule?